### ISSUE TITLE
How to use uWebsockets.js with React

### ISSUE THREAD

**Issue by totaland** [uNetworking/uWebSockets.js#620]
Hi team, 

I tried to find the solution online but there are limited resources online that show how to do this. Do you have a complete example that you use the WebSocket API (browser one) to create a client that talks to the WebSockets.js backend? 

If you have an example with React/NextJs frontend that would be awesome.

Many thanks, team

**Comment by alexhultman** [comment]
WebSockets are standardized and have no relation to or dependency on React whatsoever. 

It's like asking a car manufacturer how to shift gears using leather gloves from Levi's 1987th summer collection, as if it would be any different.

**Comment by totaland** [comment]
Yeah, the issues here are that I am quite new to this socket API and all the complete example that I found online is with socket.io instead of with uWebSocket. However, I believe uWebSocket is better than socket.io so I try to learn how to use it but it comes back to my point above that we don't have a lot of resources online for this library.

One of the main things that make NextJs so good is the investment they received as well as the numerous example they have on their GitHub page. These numerous examples make it easier to adopt the framework. I find the examples in the repo too but it is all in the backend and I have no clue at the moment how does it work and how I can try to adopt that with NextJs etc.

So maybe instead of just React let me rephrase my question, do you have a complete example from frontend to backend somewhere that I can help me learn this library a bit better. I find the docs in the repo a bit confusing... Maybe just the way I learn is a bit visual so I need some extra help.
